flict/flictlib/translator.py

```diff
@@ -12,12 +12,20 @@
 
 import argparse
 import json
-from flict.flictlib import logger
 from argparse import RawTextHelpFormatter
+try:
+    from flict.flictlib import logger as logger
+except ImportError: # allows usage also as a standalone tool
+    class MonkeyPatchLogger:
```

Comment:
That deserve some explanation, as don't quite follow the given info

Reply:
When running the file stand-alone, it would try to import logger and fail, provided the script could be executed from the same directory where it is located. There aren't many dependencies, actually none, except for the logger. By creating mocked logger class, the error would be circumvented, while preserving original functionality (code is imported in flict/flictlib/license.py, that seems to be the only usage of the code from that file)

Reply:
Is there a use case for running this file stand alone?